addons/external-player-button/addon.json

```diff
@@ -42,6 +43,12 @@
       "id": "again",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Replace the stage automatically when a recommendation in the instructions or notes is found (requires \"Replace stage\" to be turned on due to laziness)",
```

Comment:
"requires "Replace stage" to be turned on due to laziness" what does this mean?

Reply:
It means that I was lazy and I just use click() on a button to replace the stage so in case replacing the stage isn't enabled it would redirect you to fork or warp.

Reply:
Let me see if I understand, did you make a setting that depends on another setting being enabled?

Reply:
Yep, I did.

Reply:
But making that another extension wouldn't make sense.

Reply:
Also, if you want the auto replace, then you probably also want to replace the stage anyway when you click on the button.

Reply:
i disagree

Reply:
@JeffaloBob, wdym?

Reply:
There's probably a better way to do this. Please try to be a bit more professional, you know options with that name won't be merged :P

Reply:
What? Seeming unprofessional can be a professional tactic.

Reply:
>What? Seeming unprofessional can be a professional tactic.

![image](https://user-images.githubusercontent.com/40470736/92262178-05285800-eedb-11ea-9d4b-9017649b1aa5.png)

Reply:
bruh you really always have a stupid image for every situation.
No really, you can make something look unprofessional to make people think that those are a few nice guys who just fo what they think is right and you may give them all the money you have. But a big company? No, not even a quarter! They already have way too much money!

Reply:
> bruh you really always have a stupid image for every situation.
> No really, you can make something look unprofessional to make people think that those are a few nice guys who just fo what they think is right and you may give them all the money you have. But a big company? No, not even a quarter! They already have way too much money!

ever heard of discord's great customer support?

Reply:
C'mon, don't ruin a bad joke by taking it seriously.